apps/publish/enqueue/enqueue_service.py

```diff
@@ -555,6 +572,36 @@ def queue_transmission(self, doc, subscribers, subscriber_codes=None, associatio
         except Exception:
             raise
 
+    def get_unique_associations(self, associated_items):
+        associations = {}
+        for association in associated_items.values():
+            item_id = association.get("_id")
+            if item_id and item_id not in associations.keys():
+                associations[item_id] = association
+        return associations.values()
+
+    def resend_association_items(self, doc):
+        associated_items = doc.get(ASSOCIATIONS)
+        if associated_items:
+            for association in self.get_unique_associations(associated_items):
+                # resend only media association
+                if association.get("type") not in MEDIA_TYPES:
+                    continue
+
+                archive_article = get_resource_service("archive").find_one(
+                    req=None, _id=association.get("guid", association.get("_id"))
```

Comment:
it's better to use `_id` for the `_id` lookup, I think with some media we had it different so guid won't match _id